lib/stat_tracker.rb

```diff
@@ -23,10 +23,25 @@ def team_collection
 
   def game_team_collection
     GameTeamCollection.new
-  end 
+  end
 
   def highest_total_score
-
+    top_score = 0
+    CSV.foreach(@games, headers: true, header_converters: :symbol) do |game|
+      if game[:away_goals].to_i + game[:home_goals].to_i > top_score
+        top_score = game[:away_goals].to_i + game[:home_goals].to_i
+      end
+    end
+    top_score
   end
 
+  def lowest_total_score
+    lowest_score = 1_000_000_000
+    CSV.foreach(@games, headers: true, header_converters: :symbol) do |game|
+      if game[:away_goals].to_i + game[:home_goals].to_i < lowest_score
+        lowest_score = game[:away_goals].to_i + game[:home_goals].to_i
+      end
+    end
+    lowest_score
+  end
```

Comment:
Definitely learned a lot just from creating these methods- I'm sure their format will be helpful for future methods as well.